mkosi/mkosi.conf

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 [Config]
-MinimumVersion=25~devel
+MinimumVersion=fc7fb750dd3a6c0b6f0d04bfe785145799be75b6
```

Comment:
shouldn't this be something like `commit:fc7...` so that both a version and a commit can be used?

Reply:
A version can still be used. We detect a commit as a 40 length string of alphanumeric characters

Reply:
might want to make that immediately compatible with sha256